packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -54,6 +58,31 @@ abstract class FlutterTestRunner {
     String? integrationTestUserIdentifier,
     TestTimeRecorder? testTimeRecorder,
   });
+
+  /// Runs tests using the experimental strategy of spawning each test in a
+  /// separate lightweight Engine.
+  Future<int> runTestsBySpawningLightweightEngines(
```

Comment:
I've omitted some parameters of `runTests` that seemed unnecessary for `runTestsBySpawningLightweightEngines`. Are there any others that are worth omitting? Maybe `concurrency`?

Reply:
We don't need to here, but probably should file a bug to track wiring up concurrency as a parameter for e.g. how many isolates we spawn.

Reply:
I learned that `pacakge:test`'s `--concurrency` parameter correctly interracts with our `PlatformPlugin` to control the number of isolates spawned, so I have plumbed `runTestsBySpawningLightweightEngines`'s `concurrency` parameter through to `package:test` in this PR (https://github.com/flutter/flutter/pull/141726/commits/b89c1576adcccc002629686cb792fbcc8a6cab31).